docs/_documentations/mdt-intellij-getting-started.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: docs
+title: Installing Codewind on IntelliJ
+description: Installing Codewind on IntelliJ
+keywords: install, installing, IntelliJ, Eclipse, Codewind, IDE, plugin, plug-in, settings, creating, project, projects, template, code change, edit, edits, application, removing
+duration: 1 minute
+permalink: mdt-intellij-getting-started
+type: document
+---
+
+# Getting started with Codewind on IntelliJ
+Use Codewind on IntelliJ to develop your applications.
+
+## Prerequisites
+- Install [IntelliJ](https://www.jetbrains.com/idea/download/#section=mac).
+- Install Docker.
+- If you use Linux, also install Docker Compose.
+
+## Installing Codewind on IntelliJ
+1. Go to the [Eclipse Downloads page](https://download.eclipse.org/codewind/?d) for Codewind. Click **milestone** and **0.9.0**. Then, click **codewind-intellij-0.9.0.zip** to download the file.
+2. From IntelliJ, open either **Preferences** or **Settings**. A new window appears.
+   - On macOS, go to **Main**>**Preferences**.
+   - On Windows, go to **File**>**Settings**.
+3. Open **Plugins** and click the **Manage plugin repositories** icon. From the menu, click **Install Plugin from Disk...**. A window appears that displays the files on your computer.
```

Comment:
Could someone please confirm what this icon is called? It looks like a gear. The name should appear in the hover help.

Reply:
The hover help doesn't show a name, it shows a description:

![image](https://user-images.githubusercontent.com/29612372/73862624-d34bf780-480c-11ea-9be9-91ec47764387.png)

The gear icon is usually associated with settings, so it's probably called the settings icon.  However, it may be clearer to call it 'the gear icon'.